src/main/java/org/mqttbee/api/mqtt/MqttClientBuilder.java

```diff
@@ -59,17 +67,48 @@ public MqttClientBuilder forServerHost(@NotNull final String host) {
     }
 
     @NotNull
-    public MqttClientBuilder forServerPort(final int port) {
+    public MqttClientBuilder withServerPort(final int port) {
         this.serverPort = port;
         return this;
     }
 
+    @NotNull
+    public MqttClientBuilder withServerPath(final String serverPath) {
+        this.serverPath = serverPath;
+        return this;
+    }
+
     @NotNull
     public MqttClientBuilder usingSSL(final boolean usesSSl) {
         this.usesSSL = usesSSl;
         return this;
     }
 
+    @NotNull
+    public MqttClientBuilder keyStore(@Nullable final KeyStore keyStore) {
```

Comment:
Add own config for SSL: withSslConfig(MqttSslConfig)
and add a builder for this config: withKeyStore, withKeyStorePassword, withTrustStore

Reply:
done